src/Config.php

```diff
@@ -3750,4 +3750,66 @@ final public static function generateUuid($type)
         self::setConfigurationValues('core', [$type . '_uuid' => $uuid]);
         return $uuid;
     }
+
+    /**
+     * Get email and name to be used as "From" in our email notifications
+     *
+     * @param int  $entities_id
+     * @param bool $allow_response
+     *
+     * @return array [email => sender address, name => sender name]
+     */
+    public static function getSender(
```

Comment:
Maybe a less generic name would be better.
```suggestion
    public static function getAdminEmailSender(
```

Reply:
Done 7cabfbc.